components/sidebar/sidebar_channel/sidebar_channel.jsx

```diff
@@ -287,7 +287,6 @@ class SidebarChannel extends React.PureComponent {
         return (
             <li
                 key={this.props.channelName}
-                ref={'channel'}
```

Comment:
I'm not sure why this was removed?

Reply:
It doesn't look like it was used at all. There's no reference to `this.refs` in this file

Reply:
Actually, never mind, it's used by the `Sidebar` component which is really bad form.

@cometkim Can you revert this?

Reply:
Oh......... Ok. And will hunt down every legacy refs in later PR.